pkg/clients/util.go

```diff
@@ -24,12 +24,12 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/sapcc/go-pagerduty"
+	"github.com/PagerDuty/go-pagerduty"
 	"github.com/sapcc/pulsar/pkg/util"
 )
 
 // regionAlertnameRegex is used to find the region and alertname from an incident text
-const regionAlertnameRegex = `.*\s\[(?P<region>[\w-]*\w{2}-\w{2}-\d|admin|staging)\]\s(?P<alertname>.+?)\s\-.*`
+const regionAlertnameRegex = `.*[\s\*]+\[(?P<region>[\w-]*\w{2}-\w{2}-\d|admin|staging)\][\s\*]+(?P<alertname>.+?)\s\-.*`
```

Comment:
Did you run the unit tests to ensure the regex works?
Unfortunately, I travis isn't configured to run them automatically.

Reply:
Tests for parsing this from a pagerduty incident can be found here: https://github.com/sapcc/pulsar/blob/dbe87aa236280ca467d93a6a55b7730227ec0513/pkg/clients/util.go

Reply:
I did a real debug of a message - and also some regex test in a test tool / with and without "*"